task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -49,17 +49,20 @@ spec:
         chmod +x opm grpc_health_probe
 
         if ! ./opm version; then
-        echo "!FAILURE! - opm binary check failed"
+          echo "!FAILURE! - opm binary check failed"
           TESTPASSED=false
         fi
         if [ ! -f "grpc_health_probe" ]; then
-        echo "!FAILURE! - grpc_health_probe binary check failed"
+          echo "!FAILURE! - grpc_health_probe binary check failed"
           TESTPASSED=false
         fi
         if ! ./opm validate confdir; then
-        echo "!FAILURE! - opm validate check has failed"
+          echo "!FAILURE! - opm validate check has failed"
           TESTPASSED=false
         fi
+        if ! ./opm render <fbcFragmentRoot> | jq -en 'reduce (inputs | select(.schema == "olm.package")) as $obj (0; .+1) == 1'; then
```

Comment:
`<fbcFragmentRoot>`?

Reply:
Thanks! I tested it in my local but forget to replace ......

Reply:
Updated.